crashtracker/src/shared/configuration.rs

```diff
@@ -9,16 +9,18 @@ use serde::{Deserialize, Serialize};
 /// We recommend fully enabling stacktrace collection, but having an environment
 /// variable to allow downgrading the collector.
 #[repr(C)]
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Default, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub enum StacktraceCollection {
     /// Stacktrace collection occurs in the
+    #[default]
```

Comment:
Is without symbols a reasonable default? Or would users want this disabled?

Reply:
I don't really know what this option does to begin with, but it seemed reasonable to have it disabled as a default. A different value might make more sense.

Reply:
Ok, I would have assumed WithoutSymbols would be the expected default, but this is not my component.

Reply:
It's also not my component and I don't know what default would be the most reasonable, I just went with the one with value `0` as it seemed like it would have been intended to be the default but that could be wrong.

Reply:
This is fine, it is a safe default. We can get this logic reviewed by @danielsn when he is back